src/docs/src/install/unix.rst

```diff
@@ -63,11 +64,19 @@ Enabling the Apache CouchDB package repository
     echo "deb [signed-by=/usr/share/keyrings/couchdb-archive-keyring.gpg] https://apache.jfrog.io/artifactory/couchdb-deb/ ${VERSION_CODENAME} main" \
         | sudo tee /etc/apt/sources.list.d/couchdb.list >/dev/null
 
-**RedHat or CentOS**: Run the following commands::
+**RedHat(<9) or CentOS**: Run the following commands::
 
     sudo yum install -y yum-utils
     sudo yum-config-manager --add-repo https://couchdb.apache.org/repo/couchdb.repo
 
+**RedHat(>=9)**: Run the following commands::
+
+    sudo yum install -y yum-utils
+    sudo yum-config-manager --add-repo https://couchdb.apache.org/repo/couchdb.repo
+    # Enable EPEL for the SpiderMonkey dependency
+    sudo dnf config-manager --set-enabled crb
+    sudo dnf install epel-release epel-next-release
```

Comment:
For me on Almalinux 9 VM adding `epel-next-release` wasn't needed. I used just `epel-release`

Reply:
Good catch! That’s why I specifically pulled out RedHat >=9 from RedHat/CentOS <=9) — I hope that helps. IF we get reports that the “next” bit is wrong, we can adjust on the fly.

Reply:
Ah good point. It's a RHEL 9 only instructions. Most people would probably figure the Almalinux / Rocky 9 instructions. If we got a lot questions we can add them later.